### ISSUE TITLE
Call modules across github repos

### ISSUE THREAD

**Issue by ams0** [LeagueOfExtraordinaryHackers/aci-privateagents-kv-integration#12]
Perhaps we can keep modules separated from this repo and call them like I call them from here: https://github.com/ams0/terraform-templates/blob/18b11a9511fc7f041322a6d0cec50571a5fc3aab/vm/main.tf#L13

cc @jcorioland

**Comment by jcorioland** [comment]
Yes, that was my idea once the module is ready and published for the aci agents we can even call it from the Terraform registry :)
